src/docs/ganache/overview.md

```diff
@@ -12,6 +12,6 @@ layout: docs.hbs
 
 # Ganache Overview
 
-[Ganache](/ganache) is a personal blockchain for Ethereum development you can use to deploy contracts, develop your applications, and run tests.
+[Ganache](/ganache) is a personal blockchain for Ethereum and Corda development you can use to deploy contracts, develop your applications, and run tests.
 
-It is available as both a desktop application as well as a command-line tool (formerly known as the TestRPC). Ganache is available for Windows, Mac, and Linux.
\ No newline at end of file
+It is available for Ethereum and Corda as both a desktop application and also as a command-line tool for Ethereum (formerly known as the TestRPC). Ganache is available for Windows, Mac, and Linux.
```

Comment:
Just a suggestion
Ganache UI is desktop application supporting both Ethereum and Corda technology. In addition, an Ethereum version of ganache is available as a command-line tool (formerly known as the TestRPC). All versions of Ganache are available for Windows, Mac, and Linux.